### PR TITLE
[Chore #160720222] configure authors haven to use postgreSQL as db engine

### PR DESCRIPTION
**Pull request by koechkevin**
#### What does this PR do?
configures Author's Haven app to use postgreSQL as database engine.
#### Description of Task to be completed?
Upgrade from manual jwt to djangorestframework-jwt.
Add psycopg2==2.7.5 and djangorestframework-jwt==1.11.0 to requirements.txt.
update authors/settings.py by removing sqlite3 connections and adding postgreSQL connection.
Use .env file to get database details and credentials.
#### How should this be manually tested?
clone the repo, create a virtual environment and activate.
Install the dependancies
```
$ pip install -r requirements.txt
```
create a .env file
```
$ touch .env
```
Edit the .env file file with the following:
```
export DB_NAME=name of your database
export DB_USER=username you use to access the database
export DB_PASSWORD=your password
export DB_HOST=host url of your database e.g localhost
```
to use the variables,
```
$ source .env
```
Make migrations and migrate
```
$ python manage.py makemigrations
$ python manage.py migrate
```
#### Any background context you want to provide?
Authors haven uses postgreSQL as database engine. You should therefore have it installed before testing the application.
#### What are the relevant pivotal tracker stories?
#160720222

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -138,6 +142,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+       'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
```
Comment (FawazFarid) [inline authors/settings.py]: @koechkevin I'm not so familiar with both of them, can you help me understand the reason behind upgrading from manual jwt to djangorestframework-jwt?

File: authors/settings.py [hunk 2]
```diff
@@ -81,8 +81,12 @@
...
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('DB_NAME'),
+        'PASSWORD':os.getenv('DB_PASSWORD'),
+        'HOST':os.getenv('DB_HOST'),
+        'USER':os.getenv('DB_USER'),
+        'PORT':'5432',
```
Comment (FawazFarid) [inline authors/settings.py:91]: looks good to me

File: authors/settings.py (hunk 1 above)
Comment (koechkevin) [reply authors/settings.py]: We started the project with bugs. While I was debugging, I made the change to enable testing of the application.

File: authors/settings.py (hunk 2 above)
Comment (FawazFarid) [reply authors/settings.py:91]: please fix the merge conflicts

**Review by FawazFarid — APPROVED** [review]

**Pull request merged**